Player/Player.gd

```diff
@@ -181,6 +184,20 @@ func _process_movement(delta: float) -> void:
 	vel.x = hvel.x
 	vel.z = hvel.z
 	
+	if(vel.y < 0):
+		fallHeight -= 0.1
+		print(fallHeight)
+	elif (vel.y == 0 && fallHeight <= 0):
+		is_dead = true
```

Comment:
The code that's in common between this condition and the normal death check should probably be moved to a die() function.